### PR TITLE
npm tests

### PR DESCRIPTION
**Pull request by cdmistman**
we're not reading packages from the lockfile at all, which is why tests are failing

blocked on #126

### CODE REVIEW

File: internal/backends/nodejs/nodejs.go
Comment (cdmistman) [inline internal/backends/nodejs/nodejs.go:1]: this is a bugfix

File: test-suite/templates/nodejs-npm/many-deps/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,2241 @@
...
+      "license": "MIT",
+      "engines": {
+        "node": ">=10"
+      },
+      "funding": {
+        "url": "https://github.com/sponsors/sindresorhus"
+      }
+    }
+  }
+}
```
Comment (ryantm) [inline test-suite/templates/nodejs-npm/many-deps/package-lock.json:2241]: sheesh.

**Review by ryantm — APPROVED** [review]
Man those lock files for tests are big. I tried making some invalid package-lock.json files and it didn't blow up.

**Comment by cdmistman** [comment]
yeah, the test framework presumes that the lockfiles are valid, since they're an implementation detail of the package manager itself. however, i'm wary of manually editing them - things should Just Work. the only reason they're committed is to reduce lockfile resolution time. it's faster if the pm doesn't have to do any resolution

**Pull request merged**